recipes/pexsi/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "pexsi" %}
+{% set version = "1.2.0" %}
+{% set build = 0 %}
+{% set mpi = mpi or "mpich" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/berkeleylab/{{ name }}/downloads/{{ name }}_v{{ version }}.tar.gz
+  sha256: 8bfad6ec6866c6a29e1cc87fb1c17a39809795e79ede98373c8ba9a3aaf820dd
+  patches:
+    - superlu_dist-cname.patch
+    - cmake-shared-lib.patch
```

Comment:
Are these upstream in a PR or a commit? If so, please add a link/comment here. If not, are they upstreamable? Then consider sending them to the source. (Not a blocker to merge here.)

Reply:
Upstream is updating for a newer superlu_dist version already, but no release yet unfortunately. I'll stay in touch with upstream.